extensions/PhabBugz/lib/API/V1/Lando.pm

```diff
@@ -27,6 +27,7 @@ sub setup_routes {
   $r->get('/lando/uplift')->to('PhabBugz::API::V1::Lando#get');
   $r->get('/lando/uplift/:id')->to('PhabBugz::API::V1::Lando#get');
   $r->put('/lando/uplift')->to('PhabBugz::API::V1::Lando#update');
+  $r->put('/lando/uplift/:id')->to('PhabBugz::API::V1::Lando#update');
```

Comment:
Let's just support having the bug IDs in the JSON body. I've already removed it on the Lando side so having a single supported way to pass the IDs would make the code a bit simpler.